### ISSUE TITLE
"quotaPercentage is not a valid attribute" after update to 3.3.0

### ISSUE THREAD

**Issue by cahebebe** [nextcloud/mail#8791]
### Steps to reproduce

This is happening when I upgrade from 3.2.x to 3.3.0.

### Expected behavior

Show the Mail App.

### Actual behavior

I got an Internal Server Error:
"Exception":"BadFunctionCallException","Message":"quotaPercentage is not a valid attribute"

### Mail app version

3.3.0

### Mailserver or service

IRedMail 1.6.2

### Operating system

Centos 7.9

### PHP engine version

PHP 8.1

### Web server

Apache (supported)

### Database

MySQL

**Comment by thrommel1980** [comment]
Had the same issue. 
It was solved with one or a second reboot.
Then the error was not shown anymore, and the App works well.

**Comment by GretaD** [comment]
How did you try to do the upgrade, from the UI or cosole?

**Comment by cahebebe** [comment]
@GretaD I did it from UI.

**Comment by GretaD** [comment]
can you please try to reboot till we figure out why the error was shown at the first place

**Comment by MasterxAndi** [comment]
I have the same issue.
My eMail Server has been set to quota "unlimited".

If I change this to "5GB" or else, mail is working again.

**Comment by Kompromittiert93** [comment]
got the same issue. I also installed the update via UI. After a complete reboot, it works again. My quota is also "unlimited" postfix option set to "-1". So I don't think, that the value of the quota is the issue.

**Comment by MasterxAndi** [comment]
> 

A restart of PHP worked also :) ✅

**Comment by GretaD** [comment]
yes, this is a very weird behavior, im gonna keep this ticket open till we know what happened.

**Comment by ChristophWurst** [comment]
> It was solved with one or a second reboot.

Duplicate of https://github.com/nextcloud/mail/issues/7407. See also https://github.com/nextcloud/mail/issues/7737.
